app/views/avo/fields/common/_status.html.erb

```diff
@@ -0,0 +1,43 @@
+<%
+  classes = "inline-block "
+  classes += "text-red-700" if status == 'fail'
+%>
+<div class="flex flex-shrink-0">
+    <div class="<%= classes %>"><%= label %></div>
+    <% if status == 'load' %>
+      <div class="spinner ml-1"></div>
+    <% end %>
+</div>
+
+<style>
```

Comment:
We should not add style tags like this. 

I think we should place them inside `frontend/css/components/status.css` and import them into `application.css`